src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -524,7 +524,7 @@ public IObservable<bool> EnsureLocalBranchesAreMarkedAsPullRequests(ILocalReposi
 
         public bool IsPullRequestFromRepository(ILocalRepositoryModel repository, PullRequestDetailModel pullRequest)
         {
-            return pullRequest.HeadRepositoryOwner == repository.CloneUrl.Owner;
+            return string.Equals(repository?.CloneUrl.Owner, pullRequest.HeadRepositoryOwner, StringComparison.OrdinalIgnoreCase);
```

Comment:
@jcansdale 

> It's possible that repository.CloneUrl.Owner will be null if ILocalRepositoryModel doesn't have a remote.

I put the `?` here because of what you said, but is that really possible?

Reply:
It's definitely possible that `CloneUrl` is null but you put the `?` after `repository`. I don't think passing a null `repository` to this method should be allowed, so we probably need a `Guard.ArgumentNotNull(repository, nameof(repository));` at the top of the method instead of the `?`.